networking/egress-ips.html.md

```diff
@@ -52,17 +52,30 @@ fly ips list
 fly ips release-egress <ip-address>
 ```
 
+App-scoped egress IPs are only released when you explicitly run `fly ips release-egress`. They persist across Machine destruction and deployments.
+
 ### Billing
 
-Each app-scoped IPv4 static egress address costs $3.60/mo, billed hourly. IPv6 addresses are currently free, but must be allocated along with an IPv4.
+Each app-scoped IPv4 static egress address costs $3.60/mo, billed hourly. IPv6 addresses are currently free, but must be allocated along with an IPv4. During the beta period, app-scoped egress IPs are free. After the beta, app-scoped IPv4 egress IPs are expected to be billed at the same rate as machine-scoped egress IPs ($3.60/mo, billed hourly).
 
 ### Caveats
 
 - Each static egress IP can support up to 64 Machines. If you need more than 64 Machines in one region, you will need to allocate multiple static egress IPs.
-- When using App-scoped static egress IPs, a Machine can make up to 1000 connections to _each_ external IP address. There is no limit on the _total_ number of concurrent connections.
-  - We do not expect this to be a concern for most apps. However, feel free to talk to us if this limits your use case!
+- When using app-scoped static egress IPs, a Machine can make up to 1024 concurrent connections to _each_ destination IP address. There is no limit on the _total_ number of concurrent connections.
+- We do not expect this to be a concern for most apps. However, feel free to talk to us if this limits your use case!
 - When you have multiple static egress IPs assigned in one region, there is currently no way to specify exactly which IP each machine will use.
-- When new machines are created, there might be a brief window when an app-scoped egress IP is not applied to the machine. This may happen more often with more machines or during bluegreen deployment. Allocating multiple pairs of static egress IPs alleviates the issue.
+- There may be a short delay between allocating an egress IP (or creating a Machine) and the egress IP becoming usable. This is inherent to how the system applies IPs to new Machines. This delay may be more noticeable with more Machines or during bluegreen deployments. Allocating multiple pairs of static egress IPs alleviates the issue.
```

Comment:
I don't think this description is entirely accurate. It's essentially talking about two possible scenarios:

- Right after allocating a new egress IP, in that case the egress IP will be applied to all existing machines after a short delay. Allocating multiple pairs will probably _not_ help in this case (since allocating them might be slower :smile: ).
- The app already has an egress IP assigned, and a new machine is created. In that case there might be a delay before the machine gets to use the egress IP, and only in this case might allocating multiple pairs help.